workflow/rules/common.smk

```diff
@@ -57,16 +65,8 @@ def get_model(wildcards):
 
 def is_single_end(sample, unit):
     """Determine whether unit is single-end."""
-    fq2_present = pd.isnull(units.loc[(sample, unit), "fq2"])
-    if isinstance(fq2_present, pd.core.series.Series):
-        # if this is the case, get_fastqs cannot work properly
-        raise ValueError(
-            f"Multiple fq2 entries found for sample-unit combination {sample}-{unit}.\n"
-            "This is most likely due to a faulty units.tsv file, e.g. "
-            "a unit name is used twice for the same sample.\n"
-            "Try checking your units.tsv for duplicates."
-        )
```

Comment:
@tedil was it intentional to remove this check? I remember that we introduced this because multiple occurrences of the same `{sample}-{unit}` combination otherwise throws an incomprehensible error. So even if this might not be the best place to put it, we should check for this somewhere...

Reply:
True, the check should be done somewhere, probably directly after reading samples.tsv and units.tsv.
The way this check was written didn't really work anyways, since fq2_present always is a pd.Series, irrespective of the cardinality of sample-unit combinations